roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile: dest=/etc/yum/pluginconf.d/fastestmirror.conf regexp="^enabled=.*" line="enabled=0" state=present
```

Comment:
Changed to multiline. Regarding "create: yes", this would result in an invalid config as it is of INI format. I created a conditional so that lineinfile is only executed when the config exists